### DISCUSSION TITLE
How and where to put Income Tax Returns?

### DISCUSSION THREAD

**Discussion by avinal** [ananthakumaran/paisa#258] [Q&A] [answered]
Normally, I would create a new income category `Income:Tax:ITR` but that would not affect the tax calculations. ITR is supposed to be subtracted from the total tax paid. Is there a way to do that?

**Comment by ananthakumaran** [comment, marked as answer]
I don't think it's possible now. This is same as not being able to adjust refund(income) with expenses

**Reply by avinal** [reply]
Ok, thanks, I will put it as income source then.
